examples/cora.py

```diff
@@ -11,7 +11,7 @@
     T.RandomNodeSplit(num_val=500, num_test=500),
     T.TargetIndegree(),
 ])
-path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', dataset)
+path = Path.joinpath(Path(__file__).resolve().parent, '..', 'data', dataset)
```

Comment:
why not replace the `..` with a second parent?

Reply:
Yes its possible `path = Path.joinpath(Path(__file__).resolve().parent.parent, 'data', dataset)` would give the same result